proto/zitadel/webkey/v2beta/webkey_service.proto

```diff
@@ -262,17 +290,22 @@ message DeleteWebKeyRequest {
 }
 
 message DeleteWebKeyResponse {
-  zitadel.resources.object.v3alpha.Details details = 1;
-}
-
-message ListWebKeysRequest {
-  optional zitadel.object.v3alpha.Instance instance = 1 [
+  // The timestamp of the deletion of the key.
+  // Note that the deletion date is only guaranteed to be set if the deletion was successful during the request.
+  // In case the deletion occurred in a previous request, the deletion date might be empty.
+  google.protobuf.Timestamp deletion_date = 3 [
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-      default: "\"domain from HOST or :authority header\""
+      example: "\"2025-01-23T10:34:18.051Z\"";
     }
   ];
 }
 
+message ListWebKeysRequest {}
```

Comment:
Do we need options to sort and paginate?

Reply:
as of now there will be only a very few keys. typically 2 to 3.
if needed we'll add this in the future